frontend/packages/kubevirt-plugin/integration-tests-cypress/view/wizard.ts

```diff
@@ -18,18 +18,46 @@ export const wizard = {
         }
       });
     },
-    processBootSource: (source: ProvisionSource, cdrom: boolean, size: string) => {
+    processBootSource: (
```

Comment:
I don't know that using an object literal is the most maintainable way of creating our views. I think we need to look at eventually moving them to individual functions/variables or some other format. If we use individual functions/variables, things could be better organized while still calling the whole file like `import * as vmWizard from './view/wizard'`.